src/components/header/styles.js

```diff
@@ -0,0 +1,44 @@
+import styled, { css } from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const HeaderContainer = styled.header`
+  height: 100px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+`;
+
+export const Nav = styled.nav`
+  width: calc(100% - 160px);
```

Comment:
It's easier to add `padding: 0 80px` to the HeaderContainer and remove the width here. Or set it to 100%